docs/UserGuide.md

```diff
@@ -233,6 +233,13 @@ Furthermore, certain edits can cause the AddressBook to behave in unexpected way
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
 
+**Q**: What is JSON?<br>
```

Comment:
Could this be more technical than it needs to be? 

How about "How is your data stored?" - then talk about how their data is stored in a data file at (the location) and the format.

UI screenshots are good.